util/queries.js

```diff
@@ -39,9 +39,13 @@ export const FIND_COMMUNITY_BY_ADDRESS_QUERY = gql`
 export const UPDATE_COMMUNITY_DATA_HASH_MUTATION = gql`
   mutation UPDATE_COMMUNITY_DATA_HASH_MUTATION(
     $id: ID!
+    $address: String!
```

Comment:
@nicovalencia — I had to add this to get Fauna to save it. If `address` is unique and indexed, should we be using the `_id`, or conversely, this means the same address could never admin more than one community. (But perhaps that’s OK as we expect this to be a gnosis safe or other shared address?)

Reply:
Totally agree, but when I removed ID, it wouldn’t update... This might well be my lack of experience with Fauna, so if there’s some guidance on this, I’ll take it. :) I also figured it could be a followup PR to remove it.